tests/test_helper.js

```diff
@@ -0,0 +1,25 @@
+import { JSDOM } from 'jsdom';
+
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+const { window } = jsdom;
+
+function copyProps(src, target) {
+  const props = Object.getOwnPropertyNames(src)
+    .filter(prop => typeof target[prop] === 'undefined')
+    .map(prop => Object.getOwnPropertyDescriptor(src, prop));
+  Object.defineProperties(target, props);
+}
+
+global.window = window;
+global.document = window.document;
+global.navigator = {
+  userAgent: 'node.js',
+};
+copyProps(window, global);
+
+export function CustomEvent ( event, params ) {
```

Comment:
In case of default props for an argument which is an object you can also write this:
```js
export function CustomEvent(event, { bubbles = false, cancelable = false, detail = null }) { ... }
```
This way you have each of the properties as a variables inside your function, without explicitly referencing it from `params.propName`.